universal-application-tool-0.0.1/app/services/applicant/ApplicantData.java

```diff
@@ -107,6 +110,14 @@ public void putLong(Path path, String value) {
     }
   }
 
+  public void putList(Path path, ImmutableList<String> value) {
```

Comment:
Suggestion (blocking): please include javadoc that mentions the backtick being a disallowed character

Reply:
Added!